demo/custom-field-basic-demos.html

```diff
@@ -6,16 +6,44 @@
       }
     </style>
 
-    <h3>Single element wrapping</h3>
-    <vaadin-demo-snippet id="custom-field-basic-demos-sample-example" when-defined="vaadin-custom-field">
+    <h3>Creating Composite of Multiple Fields</h3>
+    <vaadin-demo-snippet id="custom-field-basic-demos-composite-example" when-defined="vaadin-custom-field">
       <template preserve-content>
-        <vaadin-custom-field label="Name">
-          <vaadin-text-field></vaadin-text-field>
+        <vaadin-custom-field label="Enter your mobile phone number">
+          <vaadin-select style="width: 100px;">
+            <template>
+              <vaadin-list-box>
+                <vaadin-item>+358</vaadin-item>
+                <vaadin-item>+46</vaadin-item>
+                <vaadin-item>+34</vaadin-item>
+              </vaadin-list-box>
+            </template>
+          </vaadin-select>
+          <vaadin-number-field></vaadin-number-field>
+          <vaadin-number-field></vaadin-number-field> 
         </vaadin-custom-field>
+        <br>
+        <vaadin-custom-field label="Choose appointment time">
+          <vaadin-date-picker></vaadin-date-picker>
+          <vaadin-time-picker></vaadin-time-picker>
+        </vaadin-custom-field>
+        <script>
+          window.addDemoReadyListener('#custom-field-basic-demos-composite-example', function(document) {
+            const phoneNumber = document.querySelectorAll('vaadin-custom-field')[0];
+            const dateTime = document.querySelectorAll('vaadin-custom-field')[1];
+            const logChanges = function(e) {
+              console.warn('change event with value: ' + e.detail.value);
```

Comment:
1. As a developer, I would expect rather `console.info` for output of this kind

2. Please consider a way to demonstrate that value contains Tab character separator

Reply:
1. Using `console.info` now
2. Using stringified value now